storage/innobase/fil/fil0fil.cc

```diff
@@ -9051,18 +9051,8 @@ dberr_t fil_tablespace_iterate(dict_table_t *table, ulint n_io_buffers,
                                            // key id used
     }
 
-<<<<<<< HEAD
     /* Check encryption is matched or not. */
     if (err == DB_SUCCESS && FSP_FLAGS_GET_ENCRYPTION(space_flags)) {
-      ut_ad(iter.m_encryption_key != NULL);
-
-||||||| merged common ancestors
-    if (FSP_FLAGS_GET_ENCRYPTION(space_flags)) {
-      ut_ad(table->encryption_key != nullptr);
-
```

Comment:
This assertion is removed because which reason?

Reply:
This assertion was incorrect, but there was no test case testing it. Now table_encrypt_4 includes one, which triggered it: it deletes the idb file of an encrypted table. The DD still has the encryption flag set, but the encryption key can't be loaded, and so it is null.